JSON/Taco Tuesday.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Author": "Raddicus",
+  "Compatibility": "Untested",
+  "DefuserDifficulty": "Easy",
+  "Description": "Set a reminder for an event by properly inputting the name of the event (Food and Day) and the day it will happen. three-displays,",
```

Comment:
Tags should be fixed here, that's incorrect (especially the comma at the end, I don't see what that's for).

Reply:
I dont understand whats wrong here.

Reply:
You want me to delete  three-displays,?

Reply:
You could do that, or you could make it like tags are supposed to be (`Tags: three-displays, buttons, numbers`).

Reply:
I got a bit confused because i opend random other json and there wasnt a Tags: think. Ok then